src/Mod/Draft/DraftEdit.py

```diff
@@ -1023,9 +1099,9 @@ def updateWire(self, obj, nodeIndex, v):
             obj.Closed = True
         # DNC: fix error message if edited point coincides with one of the existing points
         if ( editPnt in pts ) == True: # checks if point enter is equal to other, this could cause a OCC problem
-            FreeCAD.Console.PrintMessage(translate("draft", 
-                                                   "It is not possible to have two coincident points in this \
-                                                   object, please try again.")
+            App.Console.PrintMessage(translate("draft", 
+                                                   "This object does not support possible "
+                                                   + "coincident points, please try again.")
```

Comment:
You don't need a plus sign to concatenate the strings here. When there is a line break inside parentheses, the strings are concatenated automatically.
```py
"This object does not support possible "
"coincident points, please try again."
```